renovate.json

```diff
@@ -6,8 +6,8 @@
     ":gitSignOff"
   ],
   "timezone": "America/Toronto",
-  "schedule": ["on the 2nd and 4th day instance on thursday after 9pm"],
-  "enabledManagers": ["regex", "github-actions"],
+  "schedule": ["after 9pm on tuesday and thursday"],
```

Comment:
QQ: Was this a requirement from the konflux side? I'm only thinking it might increase the noise for image updates.

Reply:
I was wondering if something like https://docs.renovatebot.com/key-concepts/scheduling/#schedule-when-to-update-specific-dependencies could be of help (so we can have the `tekton` updates on a different - more often - schedule).

Reply:
@thepetk 

> QQ: Was this a requirement from the konflux side? I'm only thinking it might increase the noise for image updates.

Not a requirement from the Konflux side but due to the frequency in Konflux updates and the outdated images causing EC testing to fail, blocks deployment promotions, we need a higher frequency in renovate patching on the tekton task images.


> I was wondering if something like https://docs.renovatebot.com/key-concepts/scheduling/#schedule-when-to-update-specific-dependencies could be of help (so we can have the `tekton` updates on a different - more often - schedule).

I like this suggestion, I've opened #555 to use the Tuesday/Thursday schedule for only Konflux patching while everything else will be restored to every 2nd and 4th Thursday evening.